### ISSUE TITLE
Error in the "debounced action can be cancelled" test

### ISSUE THREAD

**Issue by dispix** [ryanseddon/redux-debounced#9]
[In the test/index.js file](https://github.com/ryanseddon/redux-debounced/blob/master/test/index.js#L180), there is a difference between the test description and the actual test. Modifying the test from `calledTwice` to `calledThrice` actually issues an error during `npm run test`.

``` javascript
it('clearTimeout is called three times', () => {
  assert.ok(global.clearTimeout.calledTwice); // Should be calledThrice according to the description
});
```

So is the description falsy or is the test actually wrong ? In the latter case, it may mean something is not working as intended.

**Comment by ryanseddon** [comment]
Think that's just an error in the text. I call `dispatch` twice which will call `clearTimeout` once and then I trigger a cancel action which will trigger `clearTimeout` again. Good catch.

**Comment by dispix** [comment]
I corrected it in my first commit in PR #10
